srcpkgs/fuzzypkg/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'fuzzypkg'
+pkgname=fuzzypkg
+version=1.0.0
+revision=1
+depends="bash"
+short_desc="XBPS fuzzy package manager"
+maintainer="Zach Dykstra <dykstra.zachary@gmail.com>"
+license="MIT"
+homepage="https://github.com/zdykstra/fuzzypkg"
+distfiles="https://github.com/zdykstra/fuzzypkg/archive/v${version}.tar.gz"
+checksum=1c62aaaf9d86dcc8bbe1bbac5b97bb18b38f6ea67f2ed8848ab2a8d2622a648f
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*|i686*|armv[67]*|aarch64*|ppc64le*)
+		depends+=" fzf"
+		;;
+	*)
+		depends+=" skim"
+		;;
+esac
```

Comment:
This is logic that's also present in the `zfsbootmenu` template, and the `void-docs-browse` one could also take advantage of it. Unless we can make virtual packages work well for cases where one of the packages isn't available on a platform (can we?), I think there should be a `void-fuzzy-finder` metapackage or similar.

Reply:
this is not resolved, as was mentioned on IRC, the dependency should be unconditionally `skim` (same in `zfsbootmenu`) and if people want `fzf`, they should install it themselves

Reply:
What is the reasoning behind that? `skim` isn't feature-for-feature compatible with `fzf`. There are distinct things that it can't do. I don't understand why the default experience for everybody should be degraded to the lowest common denominator.

Reply:
it already checks for both either way and they don't conflict when installed on the same system, so unless there is something *really* important that `fzf` can do and `skim` can't, i see no reason to make arbitrary target-specific distinctions when a portable implementation already exists

there is no degrading everybody's experience, since those who need the features can just install `fzf` manually and be done with it

Reply:
I'm not trying to be difficult here. I just haven't seen a reasonable technical argument against changing run-time depends based on availability. 

It's obviously not an issue to have both `skim` and `fzf` installed - upstream (in this case, me) prefers the usage of `fzf` where possible. Since you're now putting me in the position of having to meet someones definition of *something really important*, I don't know what else to say. I simply wanted to contribute a tool that I thought would be useful.

Reply:
presenting some good arguments *for* having the platform distinctions rather than using a single portable implementation by default would be a good start

i'm not going to be using this and have no experience with either fzf or skim, so i don't know what differences there are, that's why i'm asking you to make your case